src/components/Profile.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- Hey, please remove this eslint disabled line of code as this is not a good practice to disable the linter. Also, this would help you write clean and maintainable codes that conform to the linter. Furthermore, the line of code doesn't affect your codebase in any way, so kindly remove it.

Reply:
Removed the linter blocker.